src/node/orb.yml

```diff
@@ -8,27 +8,31 @@ executors:
     parameters:
       tag:
         type: string
+        description: Pick a specific circleci/node image variant.
```

Comment:
Would it be easy to separate the node orb changes from the above? Seems we should reason about them separately.

Reply:
@ndintenfass i needed some in here to test whether my compare-url orb is behaving as expected